doc/installation.md

```diff
@@ -1,9 +1,95 @@
 Installation
 ============
 
-```{todo}
-* explain installation for developers.
-* explain installation from conda-forge, assuming we release it there soon.
-* explain installation from PyPI, assuming we release it there soon.
-* only leave the very basics of installation in the README and refer here instead.
+The recommended way to install the svgdigitizer is to use your package manager,
+(e.g., `apt-get` on Debian or Ubuntu, `pacman` on Arch Linux, `brew` on macOS.)
+
+You can consult [repology](https://repology.org/project/svgdigitizer/packages)
```

Comment:
Currently, that's of course not terribly helpful. Let's see how quickly this project is being picked up by the distributions.

Reply:
Can we postpone this to a later moment?

Reply:
We're actually listed now:

```suggestion
You can consult [repology](https://repology.org/project/python:svgdigitizer/versions)
```